code/game/gamemodes/gang/gang.dm

```diff
@@ -0,0 +1,366 @@
+		new_gangster.my_gang = ballas
+		gangs += ballas
+		ballas.add_member(gangbanger)
+		ballas.name = new_gangster.gang_name
+
+		ballas.acceptable_clothes = new_gangster.acceptable_clothes.Copy()
+		ballas.free_clothes = new_gangster.free_clothes.Copy()
+
+		for(var/C in ballas.free_clothes)
+			var/obj/O = new C(gangbanger.current)
+			var/list/slots = list (
+				"backpack" = ITEM_SLOT_BACKPACK,
+				"left pocket" = ITEM_SLOT_LPOCKET,
+				"right pocket" = ITEM_SLOT_RPOCKET
+			)
+			var/mob/living/carbon/human/H = gangbanger.current
+			var/equipped = H.equip_in_one_of_slots(O, slots)
+			if(!equipped)
+				to_chat(gangbanger.current, "Unfortunately, you could not bring your [O] to this shift. You will need to find one.")
+				qdel(O)
+
+		gangbanger.add_antag_datum(new_gangster)
+		gangbanger.current.playsound_local(get_turf(gangbanger.current), 'sound/ambience/antag/thatshowfamiliesworks.ogg', 100, FALSE, pressure_affected = FALSE)
+		to_chat(gangbanger.current, "<B>As you're the first gangster, your uniform and spraycan are in your inventory!</B>")
+	addtimer(CALLBACK(src, .proc/announce_gang_locations), 5 MINUTES)
+	addtimer(CALLBACK(src, .proc/five_minute_warning), time_to_end - 5 MINUTES)
+	gamemode_ready = TRUE
+	SSshuttle.registerHostileEnvironment(src)
+	..()
+
+/datum/game_mode/gang/proc/announce_gang_locations()
+	var/list/readable_gang_names = list()
+	for(var/datum/team/gang/G in gangs)
+		readable_gang_names += "[G.name]"
+	var/finalized_gang_names = english_list(readable_gang_names)
+	priority_announce("Julio G coming to you live from Radio Los Spess! We've been hearing reports of gang activity on [station_name()], with the [finalized_gang_names] duking it out, looking for fresh territory and drugs to sling! Stay safe out there for the hour 'till the space cops get there, and keep it cool, yeah? Play music, not gunshots, I say. Peace out!", "Radio Los Spess", 'sound/voice/beepsky/radio.ogg')
+	sent_announcement = TRUE
+
+/datum/game_mode/gang/proc/five_minute_warning()
+	priority_announce("Julio G coming to you live from Radio Los Spess! The space cops are closing in on [station_name()] and will arrive in about 5 minutes! Better clear on out of there if you don't want to get hurt!", "Radio Los Spess", 'sound/voice/beepsky/radio.ogg')
+
+/datum/game_mode/gang/check_win()
+	var/alive_gangsters = 0
+	var/alive_cops = 0
+	for(var/datum/mind/gangbanger in gangbangers)
+		if(gangbanger.current)
+			if(!ishuman(gangbanger.current))
+				continue
+			var/mob/living/carbon/human/H = gangbanger.current
+			if(H.stat)
+				continue
+			alive_gangsters++
+	for(var/datum/mind/bacon in pigs)
+		if(bacon.current)
+			if(!ishuman(bacon.current)) // always returns false
+				continue
+			var/mob/living/carbon/human/H = bacon.current
+			if(H.stat)
+				continue
+			alive_cops++
+	if(alive_gangsters > alive_cops)
+		SSticker.mode_result = "win - gangs survived"
+		SSticker.news_report = GANG_OPERATING
+		return TRUE
+	else
```

Comment:
Pet peeve. Else after return.

Reply:
i don't understand, this proc has to return true/false

Reply:
What I meant is that you remove the else and then knock down the code behind it by one tab.